src/components/layers.js

```diff
@@ -9,7 +10,8 @@ export const Layers = {
   CAMERA_LAYER_VIDEO_TEXTURE_TARGET: 5,
 
   CAMERA_LAYER_THIRD_PERSON_ONLY: 6,
-  CAMERA_LAYER_FIRST_PERSON_ONLY: 7
+  CAMERA_LAYER_FIRST_PERSON_ONLY: 7,
+  CAMERA_LAYER_UI: 8
```

Comment:
This was introduced to simplify camera tool code, but is also something we want for other reasons in the future (namely being able to isolate UI from tonemapping after we add post effects)